### PR TITLE
Update Rahti exercices

### PR DESCRIPTION
**Pull request by trispera**

### CODE REVIEW

File: _exercises-containers/07-BasicsOfContainers.md [hunk 1]
```diff
@@ -1,7 +0,0 @@
----
-title: Basics of Containers and Orchestrations
----
-
-# Basics of Containers and Orchestrations
-
-
```
Comment (lvarin) [inline _exercises-containers/07-BasicsOfContainers.md:7]: Why was this deleted?

File: _exercises-containers/12-AppPackaging.md [hunk 2]
```diff
@@ -3,3 +3,666 @@ title: Packaging Containerized Applications
 ---
 
 # Packaging Containerized Applications
+## Helm
+[Helm](https://helm.sh/) is the "The package manager for Kubernetes", it allows the management of the life-cycle of a Kubernetes application (deploy, configure, upgrade, retire, ...). It is an **Infrastructure as Code** (IaC) tool, so it allows us to version control an application and follow its evolution along time, make identical copies (prod, preprod, dev, ...) and predictable upgrades, and of course share and publish the application. It is one of the main tools used upstream, if a tool has a "Kubernetes deployment" it is most likely going to be using Helm.
+
+## Introduction
+
+Helm packages applications into Charts. A Helm chart is a collection of `YAML` templates. In order to create a chart, one must first [install the Helm](https://helm.sh/docs/intro/install/) command line tool and install the [oc command line](/cloud/rahti/usage/cli/) tool. Once done, to continue:
```
Comment (lvarin) [inline _exercises-containers/12-AppPackaging.md]: Second link is probably broken

File: _exercises-containers/07-BasicsOfContainers.md (hunk 1 above)
Comment (trispera) [reply _exercises-containers/07-BasicsOfContainers.md:7]: I deleted it because we already have "Basics of containers" in the Lectures section.

File: _exercises-containers/12-AppPackaging.md (hunk 2 above)
Comment (trispera) [reply _exercises-containers/12-AppPackaging.md]: Yep, stupid copy paste. I'll fix
